packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -153,7 +153,7 @@ To delete all custom translations and revert to the Okta template, click **Reset
 
 ## Use Velocity Templating Language
 
-[Velocity Templating Language (VTL)](https://velocity.apache.org/engine/1.7/user-guide.html) allows you to customize your org's email templates so that you can use:
+[Velocity Templating Language (VTL)](https://velocity.apache.org/engine/2.3/user-guide.html) allows you to customize your org's email templates so that you can use:
```

Comment:
Line 170 below needs this update as well.
`In your email templates, you can use any conditional logic that VTL supports, such as `if`, `elseif`, or `else` constructs and `foreach` loops. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).`

Reply:
Nice catch! :)

Reply:
thanks, fixed